front/app/components/UI/Error/index.tsx

```diff
@@ -179,32 +172,33 @@ export type TFieldName =
   | 'category_name'
   | 'nav_bar_item_title_multiloc';
 
-export default class Error extends PureComponent<Props, State> {
-  static defaultProps: DefaultProps = {
-    marginTop: '3px',
-    marginBottom: '0px',
-    showIcon: true,
-    showBackground: true,
-    className: '',
-    animate: true,
-  };
-
-  constructor(props) {
-    super(props);
-    this.state = {
-      mounted: false,
-    };
-  }
-
-  componentDidMount() {
-    this.setState({ mounted: true });
-  }
-
-  componentWillUnmount() {
-    this.setState({ mounted: false });
-  }
+const Error = (props: Props) => {
+  const containerRef = useRef<HTMLDivElement | null>(null);
+
+  const {
+    text,
+    apiErrors,
+    fieldName,
+    marginTop = '3px',
+    marginBottom = '0px',
+    showIcon = true,
+    showBackground = true,
+    className = '',
+    animate = true,
+    id,
+  } = props;
+
+  useEffect(() => {
```

Comment:
Scroll error into view whenever it exists. This will make it much easier for users to navigate forms and understand which fields they need to change in order for the form to become valid.